docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md

```diff
@@ -7,6 +7,12 @@ ms.date: 01/23/2018
 
 # Referencing the JavaScript API for Office library from its content delivery network (CDN)
 
+> [!NOTE]
+> As an alternative to referencing the library from the CDN as described in this article, you can also get Intellisense by running the following command in a Node-enabled system prompt (or git bash window) from the root of your project folder. You must have [Node.js](https://nodejs.org) installed (which includes npm).
```

Comment:
Ricky, this is actually not correct.  It's not an "alternative to" the CDN, rather, it's an "in-addition-to" step.  It must be done within a TypeScript project to get IntelliSense.  For an out-of-the-box **JavaScript** project in VS, it doesn't need to be done (the VS project references it internally be default), but even there, it has nothing to do with the script tag on the html page.

Reply:
@Zlatkovsky My bad. Would the following be correct? 
"In addition to the steps described in this article, if you want to use TypeScript, then to get Intellisense you will need run the following … [remainder of note after "the following'  is unchanged]"